Makefile

```diff
@@ -20,14 +20,13 @@ define dl_tgz
 	fi
 endef
 
-$(GOBIN)/golangci-lint: GOLANGCI_VERSION = 1.52.2
+$(GOBIN)/golangci-lint: GOLANGCI_VERSION = 1.60.3
 $(GOBIN)/golangci-lint: Makefile
 	$(call dl_tgz,golangci-lint,https://github.com/golangci/golangci-lint/releases/download/v$(GOLANGCI_VERSION)/golangci-lint-$(GOLANGCI_VERSION)-$(GOOS)-amd64.tar.gz)
 
 .PHONY: check-golangci
 check-golangci: $(GOBIN)/golangci-lint
-	@echo $(GOPACKAGES) | sed -e 's/github.com\/scylladb\/gemini/./g' | \
-		xargs $(GOBIN)/golangci-lint run --max-issues-per-linter=0 --max-same-issues=0
+	$(GOBIN)/golangci-lint run --max-issues-per-linter=0 --max-same-issues=0
```

Comment:
I see what you want to achieve here, le'ts keep it simple, please move all the options to the `.golangci.yml` and replace whole thing with `$(GOBIN)/golangci-lint run`